<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportExportPanelWidget</class>
 <widget class="QWidget" name="ImportExportPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="importExportLoadImageButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportImportDICOMButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Import DICOM Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportSaveImageButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportSaveSceneButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Scene</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportLoadSceneButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Scene</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportSaveOverlayButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Overlay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportSaveVTKModelsButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save VTK Models</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importExportLoadOverlayButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Overlay</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
